Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -444,8 +454,17 @@ - (void)loadAdViewAdForParameters:(id<MAAdapterResponseParameters>)parameters ad
 {
     NSString *bidResponse = parameters.bidResponse;
     BOOL isBiddingAd = [bidResponse al_isValidString];
+    if ( isBiddingAd && bidResponse )
+    {
+        if ( !self.adViewbidResponseDict )
+        {
+            self.adViewbidResponseDict = [[NSMutableDictionary alloc] init];
+        }
+        self.adViewbidResponseDict[self.placementIdentifier] = bidResponse;
```

Comment:
@ManojBudumuru-vungle Tested on the device, for bidding banner, failed to load ad with the error:
```
2022-11-11 17:17:18.002401+0800 AppLovin MAX Demo App - ObjC[1526:55660] [AppLovinSdk] ERROR [ALMediationAdapterWrapper] Failed start loading [ALMediatedAdViewAd thirdPartyAdPlacementIdentifier=IAB_BANNER_1-6722317, adUnitIdentifier=55f1fdf557bea516, format=BANNER, networkName=Vungle] : *** -[__NSDictionaryM setObject:forKeyedSubscript:]: key cannot be nil
(lldb) po error
[MAError: code=-5001, message=MAX returned eligible ads from mediated networks, but all ads failed to load. Inspect -[MAError waterfall] for more info., mediatedNetworkErrorCode=-1, mediatedNetworkErrorMessage=]
```
Trough debugging, found that the `self.placementIdentifier` is `nil` when `self.adViewbidResponseDict[self.placementIdentifier] = bidResponse;`, we need to put `self.placementIdentifier = parameters.thirdPartyAdPlacementIdentifier;` before `self.adViewbidResponseDict[self.placementIdentifier] = bidResponse;`

After put `self.placementIdentifier = parameters.thirdPartyAdPlacementIdentifier;` before `self.adViewbidResponseDict[self.placementIdentifier] = bidResponse;` We can receive Bidding Banner ads:

For MREC Ads, always received sleep code. Will contact platform team side. Thanks a lot. :)

Reply:
Good catch. i forgot to remove this. we dont need it.